src/json-api-client.coffee

```diff
@@ -28,6 +28,7 @@ class JSONAPIClient extends Model
     mergeInto this, mixins
 
   beforeEveryRequest: ->
+    console.log 'using local json-api-client'
```

Comment:
will remove, keeping momentarily while try to stage branch of PFE with dependency as GitHub branch that itself has a dependency as GitHub branch.